libs/core/src/lib/menu/menu-item/menu-item.component.ts

```diff
@@ -66,7 +66,7 @@ export class MenuItemComponent implements DefaultMenuItem, OnChanges, AfterConte
     submenu: BaseSubmenu;
 
     @Output()
-    selected: EventEmitter<void> = new EventEmitter<void>();
+    onSelect: EventEmitter<void> = new EventEmitter<void>();
```

Comment:
Should it be considered as a breaking change?

Reply:
This was never emitted anywhere so I don't think so